rules/rbe_repo/container.bzl

```diff
@@ -186,6 +216,40 @@ def get_java_home(ctx, docker_tool_path, image_name):
     else:
         return None
 
+def _generate_sample_cc_project(ctx):
```

Comment:
A general refactor that helps us out a lot on Windows (especially since `docker cp` and Docker volumes do not work as we would like) is to not have the sample CC project be checked in files, but rather generated into the repository context on the fly when it is needed. We can pull this idea out into a separate PR if desired.

Reply:
I'm fine with including this in the windows support PR